### PR TITLE
617 landing page

### LINKED ISSUE #617: landing-page

### PR DESCRIPTION
**Pull request by Arfah**
#closes #617 made some copy changes to merge messaging from the other landing pages together.

### CODE REVIEW

**Comment by nikeshashar** [comment]
image

Which book do they get? CompSci or Learn Ruby in 15? There's two different messages in consecutive actions.

**Comment by Arfah** [comment]
It's the computer science one..

But the partial has become werid since we merged it.

**Comment by nikeshashar** [comment]
@Arfah I've changed the partial to say get your free e-book. Do we maybe need two partials now ? One for comp sci book and one for learn ruby in 15?

**Comment by Arfah** [comment]
No. It should just be one but the partial is now funny..

look at this issue: https://github.com/makersacademy/website/issues/619

Before it used to be transparent so  I need someone to fix this but if the copy is okay can we :shipit:

**Comment by sjmog** [comment]
@Arfah by design - the partial is now designed to be stand-alone, not included in the hero

At the design meeting we agreed it was lower priority than other things so moved it to a partial to be included towards the bottom of pages

File: source/career-changers.html.haml [hunk 1]
```diff
@@ -11,19 +11,15 @@
...
-
-%hr
-
-%section.no-padding-bottom
-  .article--centered
-    %article
-      %h3 This could be you
-      %p Our graduates come from all walks of life, everything from lawyers to teachers, accountants to estate agents to senior managers, and many, many more. They had the belief in themselves to escape an unfulfilling career, and now they get to apply their creative skills to work that they enjoy, not to mention having a better work life balance!
-      %p When was the last time you were so excited by your work that you jumped out of bed in the morning? Never?! Then quit now and make a change for the better - life’s too short not to be doing something you love.
+      %h3 Why switch to a carer as a developer?
```
Comment (pitchinvasion) [inline source/career-changers.html.haml]: career

File: source/career-changers.html.haml [hunk 2]
```diff
@@ -11,19 +11,15 @@
...
+    %p
+      Not satisfied with your job? Want to go to work and love what you do, day in day out? It’s time for a career change, and coding could be exactly what you’re looking for.
+    %p
+      At Makers Academy, we’ve helped everybody from stay-at-home parents and recruiters to scientists and lawyers make their first step towards a career they’re passionate about, in jobs with huge growth opportunities.
+    %p
+      The technology sector has been growing rapidly for years, and has been totally unaffected by the recession. With demand for developers growing 20% year-on- year, and supply of developers (mainly graduates of computer science degrees) flat lining at 1%, the demand for developers is growing rapidly. In short - the tech industry needs you!
```
Comment (pitchinvasion) [inline source/career-changers.html.haml]: probably best to downgrade it to "largely unaffected by the recession" since it looks like recession 2.0 will likely hit us thanks to Boris and Nigel

File: source/career-changers.html.haml [hunk 3]
```diff
@@ -32,7 +28,7 @@
     .centered-row
       %h4.leader Meet
       %h2 Our hiring partners
-      %p The average salary for a Makers Academy graduate as a junior developer is £32k, and after a few years experience that number jumps up to £45,000. Compare that to the UK average of £26,500 and you can see how lucrative a career coding can be.
+      %p Sounds amazing so far, right? Think about this, a Makers Academy graduate in a Junior Developer role is £32k and we work with a range of incredible companies to make sure you find an environment that suits you. Not to mention the growth opportunities and employee perks on top of that. What are you waiting for?
```
Comment (pitchinvasion) [inline source/career-changers.html.haml]: This doesn't read write, I'd reword "Think about this, a Makers Academy graduate in a Junior Developer role is £32k and we work with a range of incredible companies to make sure you find an environment that suits you."

File: source/education-leaver.html.haml [hunk 4]
```diff
@@ -31,7 +32,8 @@
     .centered-row
       %h4.leader Meet
       %h2 Our hiring partners
-      %p An average salary for a Makers Academy graduate in a junior developer role is £32k. Now compare that to the starting salary for a UK University graduate which ranges between £18,000-£24,000! While you're mates will be starting on much lower salaries, you can get ahead and earn more!
+      %p The average salary for a Makers Academy graduate in a Junior Developer role is £32k. Now compare that to the starting salary for a UK University graduate, which ranges between £18,000-£24,000. Not only will you get ahead and earn more, you’ll have great growth prospects and could be working at one of the incredible companies we’re partnered with. Chew on this, after 3 years, our graduates have paid the costs of their tuition - and made £82k on top. Meanwhile, a university graduate still has approx £25k debt. How does that measure up with your future plans?
```
Comment (pitchinvasion) [inline source/education-leaver.html.haml]: This chattier style seems different from the rest of the site, is this a deliberate experiment?

Comment (nikeshashar) [inline source/education-leaver.html.haml]: Consistency in numbers (£32k vs £18,000)

Comment (nikeshashar) [inline source/education-leaver.html.haml]: @pitchinvasion The style is taken from the CompSci booklet I believe.

**Comment by nikeshashar** [comment]
@Arfah do you still want this live?

**Comment by Arfah** [comment]
yes

File: source/career-changers.html.haml (hunk 3 above)
Comment (nikeshashar) [inline source/career-changers.html.haml]: @pitchinvasion Could you provide a suggestion as to what would sound better? Looks fine to me

**Comment by nikeshashar** [comment]
:shipit: (for now). We can make changes if needed later.

**Comment by sjmog** [comment]
I fixed the partial in #647

**Comment by danldb** [comment]
@Arfah do you want this merged?

**Comment by Arfah** [comment]
Yes @danldb
